site/docs/design_tokens/box-shadow.mdx

```diff
@@ -0,0 +1,31 @@
+---
+name: Box-shadow
+menu: Design tokens
+route: /design-tokens/box-shadow
+---
+
+import LargeParagraph from "../../src/components/LargeParagraph";
+
+# Box-shadow
+
+<LargeParagraph>Box-shadows are used to cast shadows from interface elements to backgrounds or other elements in order to allow visual cohesion and a more accessible experience.</LargeParagraph>
+
+Shadows under elements can be used to create visual distinction for content that otherwise would be hard to distinguish for the visually impaired or otherwise remain unnoticed.
```

Comment:
> Shadows under elements can be used to create visual distinction for content that otherwise would be hard to distinguish for the visually impaired or otherwise remain unnoticed.

Mielestäni tämä sisältö sopisi melkein sellaisenaan paremmin tuohon LargeParagraphiin. Tämä kuvaa paremmin varjojen merkitystä Helsingin palveluissa kuin tuo ingressi joka kuvaa varjoja yleisemmällä tasolla.